SWLOR.Game.Server/Core/NWScript/Enum/Appearance.cs

```diff
@@ -411,6 +411,7 @@ public enum AppearanceType
         // SWLOR Starts Here
 
         TiefighterSM = 870,
+        LargeShuttle = 892,
```

Comment:
Just confirming: Do these appearances already exist in our haks? Were we just missing these enumerations?

Reply:
Yes, they all exist in appearance.2da, I use them sometimes for events. I've tested the server and everything seems to spawn and function appropriately.